libraries/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/kotlin/file/EnumAsEnumBuilder.kt

```diff
@@ -79,10 +80,22 @@ internal class EnumAsEnumBuilder(
         .addKdoc("Returns all [%T] known at compile time", selfClassName)
         .maybeSuppressDeprecation(enum.values)
         .maybeAddOptIn(context.resolver, enum.values)
-        .returns(KotlinSymbols.Array.parameterizedBy(selfClassName))
+        .returns(
+            if (context.isTargetLanguageVersionAtLeast(TargetLanguage.KOTLIN_1_9)) {
```

Comment:
Not sure about this.
- Pro: consistency with `entries`
- Con: it's a breaking change

Reply:
In fact it's not that consistent, because it should be named `knownEntries` ... but that's "even more breaking" 😅

Reply:
Add `knownEntries` and deprecated `knownValues` ?